src/us_ofac.rs

```diff
@@ -4,98 +4,95 @@ use crate::{Error, Location};
 use std::collections::{HashMap, HashSet};
 
 fn get_non_passing_entries() -> HashMap<String, Option<HashSet<String>>> {
-  HashMap::from([
-    (
-      "UA".into(),
-      Some(HashSet::from(["Crimea".into()])),
-    ),
-    ("IR".into(), None),
-    ("SY".into(), None),
-    ("CU".into(), None),
-    ("KP".into(), None),
-  ])
+    HashMap::from([
+        ("UA".into(), Some(HashSet::from(["Crimea".into()]))),
```

Comment:
Optional/Equivalent: Call `into` instead of `HashSet::from`
```suggestion
        ("UA".into(), Some(["Crimea".into()].into())),
```